controllers/ramenconfig_test.go

```diff
@@ -9,16 +9,32 @@ import (
 	. "github.com/onsi/gomega"
 	ramen "github.com/ramendr/ramen/api/v1alpha1"
 	"github.com/ramendr/ramen/controllers"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/retry"
 	"sigs.k8s.io/yaml"
 )
 
 func configMapUpdate() {
 	ramenConfigYaml, err := yaml.Marshal(ramenConfig)
-	Expect(err).To(Succeed())
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
This is change is not really related, next time better split such changes to another commit (unify error handling).

Reply:
Next time I'll prepare several commits